### ISSUE TITLE
sampling points within each bin

### ISSUE THREAD

**Issue by ruinianxu** [mjq11302010044/RRPN_pytorch#12]
It seems RROI Alignment didn't use choose dynamic sampling points according to the size of bin but just one center point within each bin. Am I correct or did I misunderstand the code?

**Comment by mjq11302010044** [comment]
@ruinianxu Yep, we didn't sample the point with a dynamic rate yet. We sample only one point per bin, which is slightly different from concept of RoI Align.

**Comment by ruinianxu** [comment]
@mjq11302010044 Thanks for your reply. I have another question. If either x or y of bin center is integer, it will just bilinearly interpolate between two points. Under such case, do I need to either shift up or down or find another two nearby points? I read your code and it seems doesn't take such kind of situtation into consideration.

**Comment by mjq11302010044** [comment]
@ruinianxu Welp, it's a good question. However, first, the mentioned situation is quite rare in practical float32 computation . Second, even if it happens, I think it's okay we just interpolate between 2 pts rather than finding another nearby pts. Just as the case of both x and y are integers, we just take the values from original position on feature map.
